packages/commercetools/composables/src/getters/cartGetters.ts

```diff
@@ -0,0 +1,73 @@
+import { CartGetters, AgnosticPrice, AgnosticTotals } from '@vue-storefront/interfaces';
+import { Cart, LineItem } from './../types/GraphQL';
+import { getProductAttributes } from './productGetters';
+
+export const getCartItems = (cart: Cart): LineItem[] => {
+  if (!cart) {
+    return [];
+  }
+
+  return cart.lineItems;
+};
+
+export const getCartItemName = (product: LineItem): string => product.name;
```

Comment:
shouldn't it be `cartItem` ? its not always a product, for example in shopware it can be a discount